cloudflare/data_source_zones_test.go

```diff
@@ -2,14 +2,45 @@ package cloudflare
 
 import (
 	"fmt"
+	"log"
+	"os"
 	"strconv"
 	"testing"
 
+	"github.com/cloudflare/cloudflare-go"
 	"github.com/hashicorp/terraform/terraform"
 
 	"github.com/hashicorp/terraform/helper/resource"
 )
 
+func init() {
+	resource.AddTestSweepers("cloudflare_zones", &resource.Sweeper{
+		Name: "cloudflare_zones",
+		F:    testSweepCloudflareZones,
+	})
+}
+
+func testSweepCloudflareZones(r string) error {
+	client, _ := cloudflare.New(os.Getenv("CLOUDFLARE_TOKEN"), os.Getenv("CLOUDFLARE_EMAIL"))
```

Comment:
Nitpick: Do you mind moving this setup line into a function, like `sharedClient() error`, or even better - create a more reusable function in `cloudflare/provider.go` that can be used by both provider code and tests?

Reply:
Sure, I'll update it for the tests only as the provider auth setup needs some love and rejigging.